src/Cart/Checkout/AddressManagement/AddAddressModel/AddAddressModel.js

```diff
@@ -0,0 +1,103 @@
+import "./AddAddressModel.css";
+import { useState } from "react";
+import { Button } from "forkui-lib";
+import { IndianStates } from "./StateDatabases/IndianStates";
+import { USStates } from "./StateDatabases/USStates";
+import { AustralianStates } from "./StateDatabases/AustralianStates";
+import { Link } from "react-router-dom";
+export const AddAddressModel = ({
+  setShowAddAddressModel,
+  addresses,
+  setAddresses
+}) => {
+  const [country, setCountry] = useState("India");
+  const [state, setState] = useState("");
+  const [name, setName] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [zip, setZip] = useState(0);
+  const [mob, setMob] = useState(0);
```

Comment:
In moore than 2 states, you might need to refactor this to a useReducer to make it look cleaner

Reply:
That's better! Refactored!